MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -469,11 +469,7 @@ open class RouteLegProgress {
         var slice = legPolyline
         var accumulatedCoordinates = 0
         return Array(waypoints.drop { (waypoint) -> Bool in
-            var newSlice = slice.sliced(from: waypoint.coordinate)
-            // Work around <https://github.com/mapbox/turf-swift/pull/79>.
-            if newSlice.coordinates.count > 2 && newSlice.coordinates.last == newSlice.coordinates.dropLast().last {
```

Comment:
Just checking, this workaround is no longer needed?

Reply:
Correct. The fix was implemented in Turf by the linked PR, but since Turf itself was not updated since then, Nav SDK had to have this workaround.